### ISSUE TITLE
.markdownlint format optional extension

### ISSUE THREAD

**Issue by syntaqx** [DavidAnson/markdownlint#163]
It would be awesome if both `.markdownlint.yaml` and `.markdownlint.yml` were supported as file extensions. Because [YAML recommends](http://www.yaml.org/faq.html) `.yaml` if both files exist, it's reasonable that this should be loaded and the other ignored.

The main reasons for supporting this additional extension are:
1. Many other service tooling files specify an allowance for, preference for, or only allow for `.theirtool.yml`. (`.travis.yml`, `docker-compose.yml`, `heroku.yml`, `.golangci.yml` to name a few)
2. There is some preference (albeit not convincingly) for using 3 letter file extensions on Windows platforms.

Long story short, my repository has a bunch of these configuration files for the various tools I implement, and `.markdown.yaml` is the only one using `.yaml` as an extension. It's triggering my punctilious organizational symmetry preferences, and I need your help.

<3

**Comment by DavidAnson** [comment]
This appears to be a duplicate of https://github.com/DavidAnson/vscode-markdownlint/issues/60 which I expect to get to soon.

**Comment by syntaqx** [comment]
My apologies for the duplicate! Excited that others also would like the functionality though :) Would you like me to close this out and use the other ticket for tracking?
